jamaljsr/polar
# feat(i18n): add translations for more languages

### Description

This PR adds translations for the 10 most popular languages (according to Crowdin):
English
French
German
Spanish
Russian
Italian
Chinese Simplified
Portuguese, Brazilian
Japanese
Korean

All of the current languages, besides English, have been machine translated using Google Translate. There are sure to be many errors, but I figured it is a good enough start for anyone who wishes to help. The translations can be updated on the Crowdin project page at https://crowdin.com/project/polar

I also made the language choice persist across app restarts, which was something I've been meaning to do for a while.

### Steps to Test

1. Launch the app
2. Click on the language switcher on the bottom-right of the app
3. Click on some of the other languages
4. Confirm the text in the UI updated
5. Restart the app and confirm the previous language is used

### Screenshots

![image](https://user-images.githubusercontent.com/1356600/71292632-0eb86280-2342-11ea-81f8-c167a9a5b2e8.png)
